src/expression/node/Node.js

```diff
@@ -46,7 +47,7 @@ export const createNode = /* #__PURE__ */ factory(name, dependencies, ({ mathWit
     const context = null
 
     function evaluate (scope) {
-      const s = scope || {}
+      const s = createMap(scope || {})
```

Comment:
I don't really want to put this in here, since it's called from within the AST as well as outside.

It needs quite a lot of re-working of the tests in order to eliminate it. It is likely valuable work to be done.

Reply:
I _think_ I understand that this isn't called from within the AST directly, so I'll leave this here: i.e. it's the common entry point for evaluation, however you call it.

This would be an ideal place to call a `createInitialScope` factory method to wrap `math`, `scope` and `unit` up.